src/common/groupConfigEntityAttributes.ts

```diff
@@ -25,6 +26,10 @@ export const groupConfigEntityAttributes = {
   },
   productVersion: {
     id: 'productVersion',
-    title: 'Product Version',
+    title: 'Version',
+  },
+  product: {
+    id: 'product',
+    title: 'Product',
```

Comment:
I would rather use `productEntityAttributes` when you need it instead of adding the 'product' field here (it does not exist on the Group Config entity).